manifests/database/postgresql.pp

```diff
@@ -1,14 +1,23 @@
 # Class for creating the PuppetDB postgresql database. See README.md for more
 # information.
 class puppetdb::database::postgresql(
-  $listen_addresses  = $puppetdb::params::database_host,
-  $database_name     = $puppetdb::params::database_name,
-  $database_username = $puppetdb::params::database_username,
-  $database_password = $puppetdb::params::database_password,
-  $manage_server     = $puppetdb::params::manage_dbserver,
+  $listen_addresses     = $puppetdb::params::database_host,
+  $database_name        = $puppetdb::params::database_name,
+  $database_username    = $puppetdb::params::database_username,
+  $database_password    = $puppetdb::params::database_password,
+  $manage_server        = $puppetdb::params::manage_dbserver,
+  $manage_package_repo  = $puppetdb::params::manage_pg_repo,
+  $postgres_version     = $puppetdb::params::postgres_version,
 ) inherits puppetdb::params {
 
-  if $manage_server == true {
+  if $manage_server {
+    if $manage_package_repo {
+      class { '::postgresql::globals':
+        manage_package_repo => true,
+        version             => $postgres_version,
+        before              => Class['::postgresql::server'],
```

Comment:
as 9.4 is the newest version I thought it may be a good idea to use it. I will remove the before param